### ISSUE TITLE
Bug API RECEIVE

### ISSUE THREAD

**Issue by alexmartenexen** [blockchain/receive-payments-demos#92]
The API is working correctly, I received some payments that don't show up in my account. Below are some wallets that I RECEIVED payment, but not in my balance.

1PvMb3Zo8igeUCXFZ8gR1hQTcth6dkmeNc
14Xe2Qe7hTJycHTQ7cCw7MN5xvfsKuzvvL
1H3chn6PG9JbU83vVDLs22CzMSZtSWUQKm
16o7Jq4PCkwS9iTuQwdLGtPSSFfS2whoTq
1EbovopwKirmRandh9EkUJ2m1K6hmhPKiP
1CXX6YECZwEU7jFrUfxPWUbKmwGuddouoF
166BwUmiudUuq1iVZywAA1jvqQVUPPSwgy
14hU8aVDdKUJH9FQpVkMkMct8EvXM9wGPy
1BC3WQNnwcQMZvjwbnjUg5koTkNNm88qCz
18QyiomQzrwEtSCyBFRtBvbZFV5niXpUeY
15NAEGWY85pTupSwDuMEKERqFQmVQNP7So
19sbgm8tm9wDNM4nAzT432NQHgPasZ7io
1CkmisjvCfy2bofKsByLh4d3jBisQXsXWv
18kbG92ma5DMqGMgiGxne3kseUDWJsGgcb
1Ehqb2maa9qUq94b268zU7d8ufJFHu59U2

These are some of the wallets that my API has returned to me, customers have made payments. But it didn't enter my account.

I would like to know when you will release these payments, as these portfolios were generated by my API.

To summarize, I make the API call and it returns an address. My clients make the payments, the blockchain returns the right callback, but the money doesn't enter my account.

How to solve... I'm already with more than 3,000 dollars like that.

**Comment by asrruban** [comment]
Click Recover Funds, i think will add balance your account
